File: greensopinion.finance.ui/app/scripts/controllers/about.js
/*******************************************************************************
 * Copyright (c) 2015 David Green.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the EULA
 * which accompanies this distribution.
 *******************************************************************************/
'use strict';

/**
 * @ngdoc function
 * @name greensopinionfinanceApp.controller:AboutCtrl
 * @description
 * # AboutCtrl
 * Controller of the greensopinionfinanceApp
 */
angular.module('greensopinionfinanceApp')
  .controller('AboutCtrl',['$scope','$sce','aboutService','eulaService', function ($scope,$sce,aboutService,eulaService) {
    aboutService.about().then(function(about) {
      $scope.copyrightNotice = about.copyrightNotice;
      $scope.applicationName = about.applicationName;
    });
    eulaService.retrieveEula().then(function(eula) {
      $scope.eula = $sce.trustAsHtml(eula.text);
    });
  }]);

File: greensopinion.finance.services/src/main/java/greensopinion/finance/services/web/model/About.java
/*******************************************************************************
 * Copyright (c) 2015 David Green.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the EULA
 * which accompanies this distribution.
 *******************************************************************************/
package greensopinion.finance.services.web.model;

public class About {
	private String applicationName;
	private String copyrightNotice;

	public About() {
	}

	public About(String applicationName, String copyrightNotice) {
		this.applicationName = applicationName;
		this.copyrightNotice = copyrightNotice;
	}

	public String getApplicationName() {
		return applicationName;
	}

	public String getCopyrightNotice() {
		return copyrightNotice;
	}
}
